ipatests/test_integration/test_sssd.py

```diff
@@ -21,6 +21,7 @@ class TestSSSDWithAdTrust(IntegrationTest):
 
     topology = 'star'
     num_ad_domains = 1
+    num_clients = 1
```

Comment:
Because the test now needs 1 client, the topology needs to be updated in prci_definitions/*.yaml to use *ad_master_2client instead of *ad_master